dags/genie-nf-validate.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime
+
+from airflow.decorators import dag, task
+from airflow.models.param import Param
+
+from dag_content.nextflow_tower_functions import create_and_open_tower_workspace
+
+dag_params = {
+    "compute_env_id": Param("tJLngX5lvcm4EKhCVtAT", type="string"),
+    "pipeline": Param("Sage-Bionetworks-Workflows/nf-genie", type="string"),
+    "run_name": Param("airflow-genie-validate", type="string"),
+    "revision": Param("main", type="string"),
+    "profile": Param("aws_prod", type="string"),
+    "only_validate": Param("true", type="string"),
+    "production": Param("true", type="string"),
+    "release": Param("13.3-consortium", type="string"),
+}
+
+
+dag_config = {
+    "schedule_interval": "@daily",
```

Comment:
I like the `@daily` for now as it runs everyday at midnight.  Eventually, we may want to execute this pipeline more often.  Something like this:

```
0 9,17 * * *
```

Run twice a day at 9am and 5pm - although since I'm sure this will be UTC time, we would have to convert it to match pacific time.